### ISSUE TITLE
Terminus Manual Doc Update 

### ISSUE THREAD

**Issue by sean-e-dietrich** [pantheon-systems/documentation#2292]
Re: [Terminus Plugin Directory](https://pantheon.io/docs/terminus/plugins/directory)
## Issue Description:
Adding Terminus Upstream Deployment Plugin 
## Suggested Resolution 
Add the following plugin to the page.

https://packagist.org/packages/terminus-plugin-project/terminus-upstream-deployment-plugin

## Blocked By
- [ ] Quality standards for inclusion set by eng and documented in the Terminus guide (tracked internally

**Comment by alexfornuto** [comment]
Closing this in favor of a single issue to track all plugins awaiting entry: #2895
